client/app/pages/admin/status/status.html

```diff
@@ -43,5 +43,15 @@
         </li>
       </ul>
     </div>
+    <BR />
+    <div>
+      <ul class="list-group col-lg-4">
+        <li class="list-group-item active">Redash Database</li>
+        <li class="list-group-item" ng-repeat="(name, size) in database_metrics">
+          <span class="badge">{{size[1]}}</span>
```

Comment:
`<span>` open and closed tags don't match. You may need an open span on the next line.